_posts/2018-01-01-futures-0.3.0-alpha.3.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "Futures 0.3.0-alpha.3"
+author: "Josef Brandl"
+author_github: "MajorBreakfast"
+date:   2018-01-01
+categories: blog
+---
+
+This release introduces the `spawn!` and `spawn_with_handle!` macros for use in async functions, closures and blocks. Both macros let you spawn a task that polls the given future to completion. The task is spawned onto the current context's executor.
+
+`spawn!` lets you spawn futures with `Output = ()`:
+
+```rust
+#![feature(async_await, await_macro, futures_api)]
+#[macro_use] extern crate futures;
+
+spawn!(future).unwrap();
+```
+
+`spawn_with_handle!` lets you spawn futures with any `Output` type and also lets you await the output:
+
+```rust
+let join_handle = spawn_with_handle!(future).unwrap();
+let output = await!(join_handle);
+```
+
+`spawn!` is slightly cheaper than `spawn_with_handle!`.
```

Comment:
I think this sentence seems a bit throwaway without any explanation, it could be something really simple like

> `spawn!` is slightly cheaper than `spawn_with_handle!` since it doesn't need to provide a way to get the output.

Reply:
Sounds good.